src/allreduce_robust.cc

```diff
@@ -94,6 +94,11 @@ void AllreduceRobust::Allreduce(void *sendrecvbuf_,
     return;
   }
   bool recovered = RecoverExec(sendrecvbuf_, type_nbytes * count, 0, seq_counter);
+
+  // check if restarted worker recovered from other healthy workers
+  if (recovered) {
+      printf("[%d] all reduce with recover with seq %d\n", this->rank, seq_counter);
```

Comment:
hard to read here....[$this->rank]  recovered with seq_id $seq_counter?

Reply:
removed, will rethink how to log debug recovery in separate pr